content/2015.07.cliquet.rationale-en.rst

```diff
@@ -0,0 +1,494 @@
+Why did we make Cliquet?
```

Comment:
This is a direct translation from the original French title "Pourquoi avons-nous fait Cliquet ?" which is *passé composé*. If the previous title is incorrect then we should change both. :smile: